display.py

```diff
@@ -240,7 +240,7 @@
 #the offset is how far to the right we want to print the message.
 #initially, we want to print the message just off the display.
 offset = WIDTH
-while True:
+while true:
```

Comment:
The condition in the while loop should be `while True` with a lowercase 't' instead of `while true` to create an infinite loop.